src/components/calcite-value-list-item/calcite-value-list-item.scss

```diff
@@ -20,25 +17,24 @@ calcite-pick-list-item {
 }
 
 .handle {
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: 0 var(--calcite-spacing-quarter);
-  background-color: transparent;
-  border: none;
-  color: var(--calcite-ui-border-1);
+  @apply flex 
+    items-center 
+    justify-center 
+    py-0 
+    px-1 
+    bg-transparent 
+    border-none 
+    text-color-1
+    cursor-move;
   line-height: 0;
```

Comment:
Would `leading-none` work here?

Reply:
leading none sets it to 1, is that acceptable?

Reply:
we may be able to remove it? setting the line-height to 0 is usually a hack